server/schedule/region_scatterer.go

```diff
@@ -129,7 +130,7 @@ func NewRegionScatterer(ctx context.Context, cluster Cluster) *RegionScatterer {
 		ctx:            ctx,
 		name:           regionScatterName,
 		cluster:        cluster,
-		ordinaryEngine: newEngineContext(ctx, filter.NewOrdinaryEngineFilter(regionScatterName)),
+		ordinaryEngine: newEngineContext(ctx, filter.NewLabelConstaintFilter(regionScatterName, filter.NotTiFlashEngine, false)),
```

Comment:
How about using core.EngineTiKV

Reply:
It's different because we use `NotIn`, the store without any label will be counted. But we use `In`, we must set that label for the store.